resource_project.go

```diff
@@ -28,6 +28,42 @@ func resourceProject() *schema.Resource {
 				Required:    true,
 				Description: "Project name",
 			},
+			"ca_crt": {
```

Comment:
why not `ca_certificate`? I hate shortened names when it's not necessary :) But I'm not allergic to them either.